frontend/app/src/people/utils/nameTag.tsx

```diff
@@ -44,6 +48,61 @@ export default function NameTag(props) {
   // shorten lastSeen string
   if (lastSeen === 'a few seconds ago') lastSeen = 'just now';
 
+  if (isMobile) {
+    return (
+      <Wrap
+        isSelected={isSelected}
+        onClick={(e) => {
+          selectPerson(e);
+        }}
+        style={style}>
+        {!isSelected && (
+          <>
+            <Img src={img || `/static/person_placeholder.png`} iconSize={iconSize} />
+            <Name
+              textSize={textSize}
+              color={'#9aaec6'}
```

Comment:
do you think we can use this file for this color?

`https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/src/colors.tsx`